code/modules/mob/living/carbon/human/species_types/apid.dm

```diff
@@ -0,0 +1,98 @@
+/datum/species/apid
+	// Beepeople, god damn it. It's hip, and alive!
+	name = "Apids"
+	id = "apid"
+	say_mod = "buzzes"
+	default_color = "FFE800"
+	species_traits = list(LIPS, NOEYESPRITES)
+	inherent_biotypes = list(MOB_ORGANIC, MOB_HUMANOID, MOB_BUG)
+	mutanttongue = /obj/item/organ/tongue/bee
+	attack_verb = "slash"
+	attack_sound = 'sound/weapons/slash.ogg'
+	miss_sound = 'sound/weapons/slashmiss.ogg'
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/apid
+	liked_food = VEGETABLES | FRUIT
+	disliked_food = GROSS | DAIRY
+	toxic_food = MEAT | RAW
+	mutanteyes = /obj/item/organ/eyes/apid
+	mutantlungs = /obj/item/organ/lungs/apid
+	heatmod = 2
+	coldmod = 2
+	burnmod = 2
+	staminamod = 2
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP | SLIME_EXTRACT
+
+/datum/species/apid/random_name(gender,unique,lastname)
+	if(unique)
+		return random_unique_apid_name(gender)
+
+	var/randname = apid_name(gender)
+
+	if(lastname)
+		randname += " [lastname]"
+
+	return randname
+
+/datum/species/apid/on_species_gain(mob/living/carbon/C, datum/species/old_species, pref_load)
+	. = ..()
+	C.equip_to_slot_or_del(new /obj/item/clothing/shoes/bhop/apid(C), SLOT_NECK)
```

Comment:
The growing wings should be forced (drop current neck item or something.)

Reply:
Alright, that should fix it.